core/store/migrate/migrate.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/store/migrate/migrations" // Invoke init() functions within migrations pkg.
 )
 
-//go:embed migrations/*.sql
+//go:embed migrations/*.sql migrations/*.go
```

Comment:
Required to avoid skipping over those migrations, which does not produce an error since we use `goose.WithAllowMissing()` for legacy reasons: https://github.com/smartcontractkit/chainlink/blob/develop/core/store/migrate/migrate.go#L108-L110